apps/docs-snippets/src/guide/testing/tweaking-the-blockchain.test.ts

```diff
@@ -1,10 +1,13 @@
-import { FUEL_NETWORK_URL, Provider, fromTai64ToUnix } from 'fuels';
+import { launchNode } from '@fuel-ts/account/test-utils';
+import { Provider, fromTai64ToUnix } from 'fuels';
 
 /**
  * @group node
  */
 test('produceBlocks with custom timestamp docs snippet', async () => {
-  const provider = await Provider.create(FUEL_NETWORK_URL);
+  const { cleanup, ip, port } = await launchNode({});
+  const url = `http://${ip}:${port}/graphql`;
+  const provider = await Provider.create(url);
```

Comment:
Ok, I think this [was it](https://github.com/FuelLabs/fuels-ts/pull/1773#issuecomment-1952165919) — it turns out we had a very similar problem, solved differently:
 - https://github.com/FuelLabs/fuels-ts/pull/1590

I don't think using `launchNode` here is entirely detrimental, mainly because it fixes the problem. Still, I'd advise adding a `TODO` so we don't forget to replace this with a more definitive solution after #1356.

Reply:
Added todo: https://github.com/FuelLabs/fuels-ts/pull/1773/commits/caa1486880aa1f2335b1427d81e1befc4fbeb44d